_dasldemos/fh-cbio-intro.md

```diff
@@ -0,0 +1,247 @@
+        - Will you be uploading PHI data and if so what types?
+        - What kinds of samples do you plan to upload and approximately how many?
+        - Who will be processing the study data and uploading it to FH cBioPortal?
+        - Which group members should have access to this study upon upload?
+    - **IRB documentation details**
+        - IRB number
+        - RG number
+        - IRB Approval Letter
+        - A copy of the protocol
+The cBioPortal admin team will review your submission and notify you via email once it's been approved for upload.
+![study_teams_details](/dasldemos/assets/cbio_09_study_teams_details.png)
+![study_member_details](/dasldemos/assets/cbio_10_study_member_details.png)
+![cBioPortal_Project_details](/dasldemos/assets/cbio_11_cBioPortal_Project_details.png)
+![IRB_Details](/dasldemos/assets/cbio_12_IRB_Details.png)
+![Dataset_Details_1](/dasldemos/assets/cbio_13_Dataset_Details_1.png)
+![Dataset_Details_2](/dasldemos/assets/cbio_14_Dataset_Details_2.png)
+2. In the meantime, get [AWS credentials](/scicomputing/access_credentials/#amazon-web-services-aws) by emailing the [FH help desk](https://www.notion.so/fhdasl/helpdesk@fredhutch.org).
+*Note: Make sure to include your PI in this email request as they'll need a lab-based account as well.* Once IT emails you back with your credentials, make sure to [test them](/scicomputing/access_credentials/#testing-your-credentials) to ensure they are functioning correctly. If you already have AWS credentials, skip this step.
+
+3. Get access to the `fh-dasl-cbio` S3 bucket by emailing the cBioPortal team your AWS (Amazon Web Services) Account ID number and AWS username. Once the cBioPortal team gives you access, you will receive a confirmation email. Test your access to the cBioPortal S3 bucket by following these steps in a terminal window:
+    
+    ```bash
+    # How to test you have the correct access to the fh-dasl-cbio S3 bucket. 
+    
+    # Do the following to test if you have the correct access to the fh-dasl-cbio bucket. 
+    # You should only be able to write and list files to this S3 bucket.
+    
+    # ssh into rhino and follow the instructions here to configure AWS CLI (https://sciwiki.fredhutch.org/scicomputing/access_credentials/#configure-aws-cli)
+    
+    ssh user@rhino
+    module load awscli
+    aws configure
+    AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE 
+    AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+    Default region name [None]: us-west-2
+    Default output format [None]: 
+    
+    # You should be able to write a file into the S3 bucket 
+    
+    # Write a simple text file into the s3 bucket 
+    echo hello | aws s3 cp - s3://fh-dasl-cbio/hello.txt
+    
+    # List
+    aws s3 ls s3://fh-dasl-cbio
+    
+    # You should NOT be able to do any of these things. If you are let us know. 
+    
+    # Get object 
+    aws s3 cp s3://fh-dasl-cbio/hello.txt hello.txt
+    
+    # Delete object
```

Comment:
Maybe remove the `aws rm` example in case the user can delete files since then they might delete stuff that's not theirs, etc.

Reply:
Fair point. S3 permissions should stop them from doing that, but probably don't want to encourage them to try... hahaha. Removed.